src-electron/main-process/startup.js

```diff
@@ -192,6 +192,173 @@ async function noopConvert(resultsFile, logger) {
   }
 }
 
+/**
+ * Find all zap files in a given directory
+ * @param {*} dir
+ * @returns all .zap files in the given directory
+ */
+function findZapFiles(dir) {
```

Comment:
I thought there was already some logic that finds all the zap files in a directory? We use that for the file opening, since we only get "project directory" from UC...

Reply:
I just knew about the ${tempContentFolder} which has the .zap file for upgrade process. It is a temporary directory where the upgrade is performed. Just pass that here. Do not know about any other logic here.